source/_components/fortigate.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: page
+title: "Fortigate"
+description: "Instructions on how to integrate Fortigate Firewalls into Home Assistant."
+date: 2019-07-07 22:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: fortinet.jpg
+ha_category:
+  - Presence Detection
+ha_release: 0.95
+ha_iot_class: Local Polling
+redirect_from:
+  - /components/device_tracker.fortigate/
+---
+
+This is a Fortigate presence sensor based on device detection of the Fortigate API
+
+## Configuration
+
+Add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+fortigate:
+  host: <host_IP>
+  username: homeassistant
```

Comment:
```suggestion
  username: USERNAME
```

Reply:
Thanks, I also changed the USERNAME in the Fortigate configuration